src/main/java/com/algorand/algosdk/transaction/Transaction.java

```diff
@@ -1532,4 +1562,74 @@ public static ApplicationCallTransactionBuilder<?> ApplicationCallTransactionBui
     public static ApplicationClearTransactionBuilder<?> ApplicationClearTransactionBuilder() {
         return ApplicationClearTransactionBuilder.Builder();
     }
+
+    @JsonPropertyOrder(alphabetic = true)
```

Comment:
Does this need a `JsonInclude.Include.NON_DEFAULT` annotation to omit defaults? Not sure if that applies to nested fields.

Reply:
@algoidurovic I asked a related question in https://github.com/algorand/java-algorand-sdk/pull/340#discussion_r914143670.  Can you have a look at the linked thread and reply back?

If easier to discuss live, then let's do so.